data/maps/objects/BillsHouse.asm

```diff
@@ -1,3 +1,8 @@
+	object_const_def
+	const BILLSHOUSE_BILL_POKEMON
```

Comment:
```suggestion
	const BILLSHOUSE_MONSTER
```

Reply:
`BILLSHOUSE_BILL_POKEMON` is good with me.